ansible/files/gazebo/models/ground_plane/model-1_2.sdf

```diff
@@ -0,0 +1,39 @@
+<?xml version="1.0"?>
```

Comment:
Please check this script https://github.com/shadow-robot/sr-build-tools/blob/master/docker/utils/load_gazebo_models.sh

Reply:
That method gets all gazebo models - the one in this PR just gets two SDFs, sun and ground_plane Do we want all of the models in our images? Or should I wget just those two models?

Reply:
Well it will be few megabytes more but it will save start up time. So I will prefer to have everything on disk